File: java/Calculator.java
class Calculator {
    public static double add(double x, double y) {
        // Add two numbers
        return x + y;
    }

    public static double subtract(double x, double y) {
        // Subtract y from x
        return x - y;
    }

    public static double multiply(double x, double y) {
        // Multiply two numbers
        return x * y;
    }

    public static String divide(double x, double y) {
        if (y != 0) {
            // Divide x by y, handle division by zero
            return Double.toString(x / y);
        } else {
            return "Error: Division by zero";
        }
    }
}

public class Main {
    public static void main(String[] args) {
        Calculator calculator = new Calculator();

        double x = 10;
        double y = 5;

        double resultAdd = calculator.add(x, y);
        double resultSubtract = calculator.subtract(x, y);
        double resultMultiply = calculator.multiply(x, y);
        String resultDivide = calculator.divide(x, y);

        System.out.println(x + " + " + y + " = " + resultAdd);
        System.out.println(x + " - " + y + " = " + resultSubtract);
        System.out.println(x + " * " + y + " = " + resultMultiply);
        System.out.println(x + " / " + y + " = " + resultDivide);
    }
}

File: python/Calculator.py
class Calculator:
    @staticmethod
    def add(x, y):
        # Add two numbers
        return x + y

    @staticmethod
    def subtract(x, y):
        # Subtract y from x
        return x - y

    @staticmethod
    def multiply(x, y):
        # Multiply two numbers
        return x * y

    @staticmethod
    def divide(x, y):
        if y != 0:
            # Divide x by y, handle division by zero
            return x / y
        else:
            return "Error: Division by zero"

# Example usage of the calculator:
calculator = Calculator()

x = 10
y = 5

result_add = calculator.add(x, y)
result_subtract = calculator.subtract(x, y)
result_multiply = calculator.multiply(x, y)
result_divide = calculator.divide(x, y)

print(f"{x} + {y} = {result_add}")
print(f"{x} - {y} = {result_subtract}")
print(f"{x} * {y} = {result_multiply}")
print(f"{x} / {y} = {result_divide}")
